xbmc/music/Song.h

```diff
@@ -72,6 +72,11 @@ class CSong: public ISerializable
     return false;
   }
 
+  //Get artist names as a string vector from the vector of artist credits
```

Comment:
Please do it again. Their named doxygen comments, it's all about helping someone else understand whats going on faster.
See: http://kodi.wiki/view/HOW-TO:Submit_a_patch#Code_documentation